src/biz/metrics.go

```diff
@@ -164,15 +164,13 @@ func (mc *MetricsFetcher) cpu(metrics raw.Metrics, json *types.ContainerJSON) CP
 
 	cpu := CPU{}
 
-	if metrics.CPU.OnlineCPUs != 0 {
-		cpu.LimitCores = float64(metrics.CPU.OnlineCPUs)
-	} else { // fallback if OnlineCPUs info is not available
+	// Set LimitCores to first honor CPU quota if any; otherwise try to set it from OnlineCPUs.
```

Comment:
```suggestion
```

The behaviour was changed but the comment was not, maybe we can get rid of it now, WDYT?